### ISSUE TITLE
How to parse string 'x_2' into variable x_2 ?

### ISSUE THREAD

**Issue by proy87** [sympy/sympy#11229]
I use parse_expr('x_2') to convert a string into symbol, but without luck.
It is converted into 2x.
It appears that parse_expr returns _
How to avoid this?

**Comment by smichr** [comment]
Do these work for your purposes?

```python
>>> var('x_2')
x_2
>>> symbols('x_2')
x_2
```

**Comment by kritkaran94** [comment]
@proy87 I guess you should use symbols('string' ) to  convert a string into symbol. It would work fine.

**Comment by proy87** [comment]
@smichr, in general I need to parse arbitrary input like sin(x_2)+exp(t)/t
I know that symbol works, but I need parse_expr

**Comment by kritkaran94** [comment]
@proy87 What output do you expect for the same? 
![screen shot 2016-06-11 at 12 35 53 pm](https://cloud.githubusercontent.com/assets/10063868/15983820/36c08cca-2fd1-11e6-8105-c70697d67e63.png)

**Comment by proy87** [comment]
@kritkaran94 , you're right.
It works.

**Comment by kritkaran94** [comment]
resolved #11229

**Comment by kritkaran94** [comment]
fixed #11229

**Comment by smichr** [comment]
just for completeness

```python
>>> from sympy.parsing.sympy_parser import parse_expr
>>> parse_expr('x_2')
x_2
>>> type(_)
<class 'sympy.core.symbol.Symbol'>
```

So I will close this issue -- please reopen if there is something that is not working as you believe it should.
